mlos_bench/mlos_bench/run.py

```diff
@@ -108,7 +108,7 @@ def _optimize(*,
             (configs, scores, status) = exp.load()
             opt_context.bulk_register(configs, scores, status)
             # Complete any pending trials.
-            for trial in exp.pending_trials():
+            for trial in exp.pending_trials(datetime.utcnow(), running=True):
```

Comment:
Future work: instead of simple loop, also support a class that implements a schedule (order) on the available pending trials.

That class should be swappable/configurable, to allow us to explore different scheduling policies.

Reply:
yes - for starters, I was thinking about an "ordering" functor that would look simply like this:
```python
for trial in scheduling_policy(exp.pending_trials(datetime.utcnow(), running=True)):
    # ...
```